release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1930,6 +1935,45 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, when creating a Container-Native Storage API (CAPI) cluster on installer-provisioned infrastructure for {ibm-power-server-name}, the load balancer would become busy and stall. With this release, you can use the 'AddIPToLoadBalancerPool' command in a `PollUntilContextCancel' loop to restart the load balancer. (link: https://issues.redhat.com/browse/OCPBUGS-35088[*OCPBUGS-35088*])
+
+* Previously, an installer-provisioned installation on a bare-metal platform when Federal Information Processing Standards (FIPS) nodes are enabled caused installation failures. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34985[*OCPBUGS-34985*])
+
+* Previously, when creating an install configuration for an installer-provisioned installation on {ibm-power-server-name}, the survey stopped if the administrator did not enter a command on the {oc-first}. The survey stopped because no default region was set in the `install-config` survey. With this release, the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-34728[*OCPBUGS-34728*])
+
+* Previously, Solid state drives (SSD) using Serial ATA Solid State Drive (SATA) hardware were identified as removable. The Assisted Installer for {product-title} reported that no eligible disks were found and the installation stopped. With this release, removable disks are eligible for installation. (link:https://issues.redhat.com/browse/OCPBUGS-34652[*OCPBUGS-34652*])
+
+* Previously, Agent-based installations with dual-stack networking failed due to IPv6 connectivity check failures, even though IPv6 connectivity could be established between nodes. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31631[*OCPBUGS-31631*])
+
+* Previously, due to a programming error, a script created compute server groups with the policy set for control planes. As a consequence, the 'serverGroupPolicy' property of `install-config.yaml` files was ignored for compute groups. With this fix, the server group policy set in the `install-config.yaml` file for compute machine pools is applied at installation in the script flow. (link:https://issues.redhat.com/browse/OCPBUGS-31050[*OCPBUGS-31050*])
+
+* Previously, when configuring an Agent-based installation that uses the `openshift-baremetal-install` binary, the installer erroneously attempted to verify the libvirt network interfaces. This might cause the following error: “Platform.BareMetal.externalBridge: Invalid value: "baremetal": could not find interface "baremetal"”. With this update, as the Agent-based installation method does not require libvirt, this erroneous validation has been disabled and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30941[*OCPBUGS-30941*])
```

Comment:
```suggestion
* Previously, when configuring an Agent-based installation that uses the `openshift-baremetal-install` binary, the installer erroneously attempted to verify the libvirt network interfaces. This might cause the following error: `Platform.BareMetal.externalBridge: Invalid value: "baremetal": could not find interface "baremetal"`. With this update, as the Agent-based installation method does not require libvirt, this erroneous validation has been disabled and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30941[*OCPBUGS-30941*])
```

or

```suggestion
* Previously, when configuring an Agent-based installation that uses the `openshift-baremetal-install` binary, the installer erroneously attempted to verify the libvirt network interfaces. This might cause the following error:
+
[source,text]
----
Platform.BareMetal.externalBridge: Invalid value: "baremetal": could not find interface "baremetal"
----
+
With this update, as the Agent-based installation method does not require libvirt, this erroneous validation has been disabled and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30941[*OCPBUGS-30941*])
```

Reply:
:heavy_check_mark: